### PR TITLE
Add travis deploy

### PR DESCRIPTION
**Pull request by joostdevries**

### CODE REVIEW

File: scripts/travis-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Don't deploy on PRs
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+  exit 0
+fi
+
+if [ "$TRAVIS_BRANCH" == "master" ]; then
+  # Deploy to staging on a merge to master
+  ember deploy -e staging
+elif [ "$TRAVIS_TAG" != "false" ]; then
```
Comment (rwjblue) [inline scripts/travis-deploy.sh]: I think the order might need to be swapped here (check for the tag first, then master).  I can't recall off the top of my head what the branch is for tag builds...

Comment (rwjblue) [inline scripts/travis-deploy.sh]: Nevermind, apparently `TRAVIS_BRANCH` is not set on the build running for a tag.

Comment (joostdevries) [inline scripts/travis-deploy.sh]: is that a "+1" :trollface:

Comment (rwjblue) [inline scripts/travis-deploy.sh]: Yes, this is good, no need to tweak.

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ---
+AWS_ACCESS_KEY_ID: b1ugrISCMawg4LDOEnj91Eu3eb8EnxwOD63Y9owfxdaclmp/K4wOUdRXLE8Gmv0FXbzPJXbMld6a4wx0qfq95pQMy22gut0o4e9IAxPguhIeF9wFzZdw2nQFTEV2+T4Y52QIhPXnV403bt4hZZbIzfniwVRsdHoac60t9NoyBOI=
+AWS_ACCESS_KEY_SECRET: X1tocl6D1nwzVtUjlR9n/PbtkhtgyGjmK0qUyL5DeTX8bY6pcIMZU3g5S1yoseMzV3DnjyoRrO1vZibFFRTPPa2iDVtMAjuVyKeQVhkMl++hslcF4QsO/qlFJJg2B2twlanKQDmiRG14dM032wyQcbHLQKVFW6yF6XoW4ZTUYBk=
+
```
Comment (Gaurav0) [inline .travis.yml]: DO NOT COMMIT YOUR KEYS

CHANGE THEM NOW

Comment (rwjblue) [inline .travis.yml]: These types of keys are typically encrypted, which may be the case here (they are not generally this long when unencrypted).

@joostdevries can confirm (or fix :dancer:)...

Comment (joostdevries) [inline .travis.yml]: very :lock:. no worries.

Comment (joostdevries) [inline .travis.yml]: aka encrypted using travis public key

**Pull request merged**